.github/workflows/march-image-build-push.yml

```diff
@@ -0,0 +1,33 @@
+name: "Build and Push Multi-Arch Image"
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+      - "demo-*"
+    tags:
+      - "v*"
+
+concurrency:
+  group: march-build-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  push-quay:
+    name: Build and Push Manifest
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Build Operator Container
```

Comment:
Awesome this is copy/paste error, but not an operator. Looks good other than that mega nit

Reply:
Doh! Thank you for catching that!